additional changes to the welcome/readme

gilad-shaham opened the pull request.

adiso75 commented:
the link for the spark demo doesn't work https://github.com/v3io/tutorials/blob/a7c066f08a90fbeaeed7b5a5a251975b1468b2cc/demos/spark/mlrun_sparkk8s.ipynb

also, under the tutorial section let's add another item "5 - click on the Jupyter service"

gilad-shaham commented:
> 
> 
> the link for the spark demo doesn't work https://github.com/v3io/tutorials/blob/a7c066f08a90fbeaeed7b5a5a251975b1468b2cc/demos/spark/mlrun_sparkk8s.ipynb
> 
> also, under the tutorial section let's add another item "5 - click on the Jupyter service"

I removed the link to spark for now (it will take time until we have that demo, that repo doesn't exist).
Added the 5th step you mentioned.

Sharon-iguazio commented:
The relevant changes from this PR were incorporated into PR #217 and merged to `development`.
